linode/manager
# refactor: Finish organizing components

## Description 📝
- This PR's goal is to finish deprecating the `src/components/core` folder to bring consistency to how we consume components 🎉 📁 
  - This means we can finally just trust `src/components/[name]` as the **single source of truth** 🥳
- I know these PRs might be getting annoying but this should be the last one!! 

## Preview 📷 

### Before 😵‍💫

```ts
import React, { useEffect, useState } from 'react';

import { Notice } from 'src/components/Notice/Notice';
import { Typography } from 'src/components/Typography';
import { Paper } from 'src/components/Paper';
import { Tab } from 'src/components/core/ReachTab';
import { TabList } from 'src/components/core/ReachTabList';
import TabPanel from 'src/components/core/ReachTabPanel';
import TabPanels from 'src/components/core/ReachTabPanels';
import Tabs from 'src/components/core/ReachTabs';

export interface Tab {
...
```

### After 🧼

```ts
import React, { useEffect, useState } from 'react';

import { Notice } from 'src/components/Notice/Notice';
import { Paper } from 'src/components/Paper';
import { Tab } from 'src/components/ReachTab';
import { TabList } from 'src/components/ReachTabList';
import { TabPanel } from 'src/components/ReachTabPanel';
import { TabPanels } from 'src/components/ReachTabPanels';
import { Tabs } from 'src/components/ReachTabs';
import { Typography } from 'src/components/Typography';

export interface Tab {
...
```

## How to test 🧪

> [!important]
> There shouldn't be much manual testing needed. This PR mostly just updates imports and file names.

- Verify automated checks pass
- Glance over the diff
- Verify Cloud Manager works as expected
